apps/api/src/app/controllers/command/__snapshots__/command-payload-schemas.spec.ts.snap

```diff
@@ -107,43 +137,59 @@ exports[`command payload schemas The schema for command CREATE_COURT_CASE_BIBLIO
 Object {
   "abstract": Object {
     "coscradDataType": "NON_EMPTY_STRING",
+    "description": "a brief summary of the court case",
     "isArray": false,
     "isOptional": true,
+    "label": "abstract",
   },
   "caseName": Object {
     "coscradDataType": "NON_EMPTY_STRING",
+    "description": "the name of the court case",
     "isArray": false,
     "isOptional": false,
+    "label": "case name",
   },
   "court": Object {
     "coscradDataType": "NON_EMPTY_STRING",
+    "description": "name of the court in which the case was decided",
     "isArray": false,
     "isOptional": true,
+    "label": "court",
   },
   "dateDecided": Object {
     "coscradDataType": "NON_EMPTY_STRING",
+    "description": "the date that the court case was decided",
     "isArray": false,
     "isOptional": true,
+    "label": "date decided",
   },
   "id": Object {
     "coscradDataType": "UUID",
+    "description": "a unique identifier for the new court case bibliographic reference",
     "isArray": false,
     "isOptional": false,
+    "label": "ID (generated)",
```

Comment:
Ideally, we would automatically append the 'generated' based on knowledge that the command is index-scoped, the field is a UUID, and the field does not refer to another aggregate.